### ISSUE TITLE
F.softmax(logits)  and CrossEntropyLoss() is right?

### ISSUE THREAD

**Issue by alphanlp** [yongjincho/cnn-text-classification-pytorch#1]
torch.nn.CrossEntropyLoss(weight=None, size_average=None, ignore_index=-100, reduce=None, reduction='mean')[SOURCE]
This criterion combines nn.LogSoftmax() and nn.NLLLoss() in one single class.  I guess  F.softmax(logits)  and CrossEntropyLoss() used together is wrong?
